shared/reducers/wallets.tsx

```diff
@@ -439,6 +439,10 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
       )
     case WalletsGen.clearTrustlineSearchResults:
       return state.update('trustline', trustline => trustline.set('searchingAssets', undefined))
+    case WalletsGen.showAdvancedSendFormReceived:
+      return state.merge({
+        building: state.get('building').merge({shouldShowAdvancedSendForm: action.payload.shouldShow}),
```

Comment:
I can see an argument that the bool should be going into `built` (data received by the service) instead of `building` (data the frontend is preparing to send down to the service to be built), what do you think?